app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/UpscanConnector.scala

```diff
@@ -279,6 +282,21 @@ class UpscanConnectorImpl @Inject() (
     implicit hc: HeaderCarrier
   ): EitherT[Future, Error, Unit] = {
 
+    implicit val actorSystem       = ActorSystem()
```

Comment:
won't this start a whole new `ActorSystem`? If you need a materializer I think you might be able to inject one in to the class. If you can't you should be able to inject in the actor system so you don't start a whole new one every time this method is called